wiki/docs/en/onboarding/index.md

```diff
@@ -2,42 +2,49 @@
 
 ## CEDILLE philosophy
 
-- Linux
-- Open-source
-- Git
-- DevOps
-- IaC
+We aim to bridge the gab between the academic world and the industry by making knowledge and technology accessible to all. We promote open-source, collaboration, and Infrastructure as Code to ensure transparency and simplicity. With a DevOps approach, we encourage continuous experimentation and collective growth.
 
-We explain why we value these concepts and how they are integrated into our
-daily work.
+Our club is a space where a passion for technology allows every member to participate, learn, and build concrete projects that serve the student community at ÉTS. We train future engineers to take on real-world challenges, with a drive to innovate and transform the industry today.
 
 ## Our standards
 
 - Code quality
 - Documentation
 - Communication
-- Git workflow (issue title and description, branch naming, commit message, pull
-  request title and description, etc. )
+- Git workflow (issue title and description, branch naming, commit message, pull request title and description, etc. )
 
 ## Domain knowledge
-
-Insert links into each categories with relevant resources.
+Explore the following resources for a solid introduction to the key technologies used at CEDILLE. These links offer foundational knowledge that will be helpful for the hands-on labs.
 
 ### App development
+* Python
+  * [Official Documentation](https://docs.python.org/3/tutorial/index.html)
+  * [Python Examples](https://www.w3schools.com/python/python_examples.asp)
+
+* Go
+  * [Official Documentation](https://go.dev/doc/)
+  * [Go by Example](https://gobyexample.com/)
 
 ### Web development
+* Grav
+  * [Official Documentation](https://learn.getgrav.org/17)
+* Hugo
+  * [Official Documentation](https://gohugo.io/documentation/)
 
 ### DevOps
+* [What is DevOps?](https://github.com/resources/articles/devops/what-is-devops)
+* [DevSecOps Guidelines](https://owasp.org/www-project-devsecops-guideline/latest/00a-Overview)
+* [GitHub Actions](https://docs.github.com/en/actions)
+* [The DevOps Handbook](https://books.google.ca/books/about/The_DevOps_Handbook.html?id=8kRDEAAAQBAJ&redir_esc=y)
+* [The Unicorn Project](https://www.google.ca/books/edition/The_Unicorn_Project/kNSSDwAAQBAJ?hl=en&gbpv=1&printsec=frontcover)
+* [The Phoenix Project](https://books.google.ca/books/about/The_Phoenix_Project.html?id=mqXomAEACAAJ&redir_esc=y)
 
 ### SRE (Site Reliability Engineering)
+* [Google's Site Reliability Engineering book](https://sre.google/sre-book/table-of-contents/)
 
-## Onboarding tracks
-
-Now that you have a good understanding of our philosophy, standards, and domain
-knowledge, you can choose a track that fits your profile. Head to the next
-section to see the available tracks.
+## Hands-on labs
+It's time to put into practice what you've just learnt! Click [here](./hands-on-labs/index.md) to get started with the Hands-on labs.
```

Comment:
I have mixed feeling about this closing thought.
The hands on lab is where the learning takes place 😂
Shouldn't we close this page with something more along the lines of :

> # Moving Forward
> 
> Now that you have a clearer understanding of our community's mission and philosophy, it's time to dive in and embark on your first project. In the spirit of our club's mission—and to kickstart your journey toward becoming a fully-fledged engineer—we've crafted a hands-on lab where you'll build your own personal portfolio website and launch it live on the web.
> 
> Through this lab, you will be able to learn the basics of everything you will need to successfully start your journey with CEDILLE. But even more importantly, you will create your own personal tool to start documenting your accomplishments and setting the foundation for your future professional success.
> 
> Click [here](https://github.com/ClubCedille/Plateforme-Cedille/pull/261/hands-on-labs/index.md) to get started with the Hands-on Labs.

Or really any other closing thought as long as it 
- acknowledge that they now understand the mission of the community they are joining
- introduces the next part of the onboarding process as a personal investment. This isn't going to be some random project that they are going to be making for a cause they have no personal attachment to yet. but rather it is a first step in investing on themselves and building the tool that will allow them to (hopefully) start documenting their accomplishment on their road to success.